docs/core/testing/unit-testing-mstest-runner-intro.md

```diff
@@ -87,146 +99,143 @@ The `dotnet run` command can be used to build and run your test project. This is
+#### Debug the app directly in Visual Studio
 
-1. Navigate the test project you want to run in Solution Explorer, right click it and select `Set as Startup Project`.
+`Testing.Platform` test project can be debugged directly. To debug all the tests in the given executable, unless a filter is provided:
 
+1. Navigate the test project you want to run in Solution Explorer, right select it and select **Set as Startup Project**.
 1. Set breakpoint into the test that you'd like to debug.
+1. Go to **Debug** > **Start Debugging** (or press <kbd>F5</kbd>) to debug the selected test project.
 
-1. Go to `Debug > Start Debugging`, (or press `F5`) to debug the selected test project.
-
-All tests will be executed until your test with a breakpoint is reached. Step through your test to debug it. Once you are done debugging the application will resume running all remaining tests, unless you stop it.
+All tests are executed until your test with a breakpoint is reached. Step through your test to debug it. Once you're done debugging the app resumes running all remaining tests, unless you stop it.
 
-### Using Test Explorer
+### Use Test Explorer
 
-Testing.Platform tests partially integrate with Test Explorer. Tests can be Run and Debugged from Test Explorer.
+`Testing.Platform` tests partially integrate with Visual Studio's **Test Explorer**. Tests can be run and debugged from **Test Explorer**.
 
-> [!IMPORTANT]
-> Automatic update of tests without building the project is not yet available.
-
-To run a test, navigate to Test Explorer, select the test (or tests) to run. Right click it, and choose `Run`.
+To run a test, navigate to **Test Explorer**, select the test (or tests) to run. Right select it, and choose **Run**. Similarly to debug a test, select the test (or tests) in **Test Explorer**, right select and choose **Debug**.
 
-Similarly to debug a test, select the test (or tests) in Test Explorer, right click and choose `Debug`.
+> [!TIP]
+> Automatic update of tests without building the project isn't available.
 
-## In Continuous Integration (CI)
+## In continuous integration (CI)
 
-There is no special pipeline task, or any additional tooling to run Testing.Platform tests. There is also no additional tooling to run multiple tests projects through a single command.
+There's no special pipeline task, or any extra tooling to run `Testing.Platform` tests. There's also no other tooling required to run multiple tests projects through a single command.
 
-To run a test project in CI add one step for each test executable that you wish to run, such as this on AzureDevOps:
+To run a test project in CI add one step for each test executable that you wish to run, such as this on Azure DevOps:
 
 ```yml
 - task: CmdLine@2
-  displayName: "Run Contonso.MyTests.exe"
+  displayName: "Run Contoso.MyTests.exe"
   inputs:
     script: '.\Contoso.MyTests\bin\Debug\net8.0\Contoso.MyTests.exe'
 ```
 
 ## Extensions
```

Comment:
This should be put after the table or I think it would be even better in a separate extension page.

Reply:
Yes, that's what I fixed when I saw the preview. Good catch.